templates/kindergarden/kindergarten_page.html

```diff
@@ -0,0 +1,56 @@
+{% extends 'base.html' %}
+{% load static %}
+
+{% block header %}
+    {{ block.super }}
+        <link href="{% static 'final_project/src/css/kinder_list.css' %}" rel="stylesheet">
+
+{% endblock %}
+
+{% block main %}
+<body>
+
+    <div class="navbar navbar-inverse bg-inverse">
+      <div class="container d-flex justify-content-between">
+        <a href="#" class="navbar-brand">Головна</a>
+        <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarHeader" aria-controls="navbarHeader" aria-expanded="false" aria-label="Toggle navigation">
+          <span class="navbar-toggler-icon"></span>
+        </button>
+      </div>
+    </div>
+
+    <section class="jumbotron text-center">
+      <div class="container">
+        <h1 class="jumbotron-heading">Список дитячих садків</h1>
+        <p class="lead text-muted"> Тут ви можете обрати дитячий садок який найбільше підходить до ваших вподобань, і щасливого вам дня. </p>
+
+      </div>
+    </section>
+
+    <div class="album text-muted">
+      <div class="container">
+          <div class="row">
+              {% for kinder in kinder_list%}
+                 {{ district_object }}
```

Comment:
Do you have district object, really?
If you want to get some data from district, you should `kinder.district.name` for example